x-pack/plugins/observability_solution/infra/public/plugin.ts

```diff
@@ -44,7 +44,6 @@ import type {
   InfraClientStartExports,
 } from './types';
 import { getLogsHasDataFetcher, getLogsOverviewDataFetcher } from './utils/logs_overview_fetchers';
-
```

Comment:
Q: Is the space here removed intentionally?

Reply:
No, probably I deleted the space when removing some import I'm not using anymore